TranslatorReasonerAPI.yaml

```diff
@@ -1022,11 +1024,48 @@ components:
           items:
             $ref: '#/components/schemas/MetaAttribute'
           nullable: true
+        qualifiers:
+          description: >-
+            Qualifiers that are possible to be found on this edge type.
+          items:
+            $ref: '#/components/schemas/MetaQualifier'
+          nullable: true
+          type: array
+        association:
+          type: object
+          $ref: '#/components/schemas/BiolinkEntity'
+          description: >-
+            The Biolink association type (entity) that this edge represents.
+            Associations are classes in Biolink
+            that represent a relationship between two entities.
+            For example, the association 'gene interacts with gene'
+            is represented by the Biolink class,
+            'biolink:GeneToGeneAssociation'.  If association
+            is filled out, then the testing harness can
+            help validate that the qualifiers are being used
+            correctly.
+          example: biolink:ChemicalToGeneAssociation
       required:
         - subject
         - predicate
         - object
       additionalProperties: false
+    MetaQualifier:
+      type: object
+      properties:
+        qualifier_type_id:
+          $ref: '#/components/schemas/CURIE'
+          description: >-
+            The CURIE of the qualifier type.
+          example: biolink:subject_aspect_qualifier
+        applicable_values:
+          type: array
+          description: >-
+                The list of values that are possible for this qualifier.
+          items:
+            type: string
+            example: [expression, activity, abundance, degradation]
+            nullable: true
```

Comment:
This nullable:true seems to be nested under items? Implying that null items can be in the list? Perhaps this should be de-indented so that applicable_values is the nullable:true thing?